iptables/iptables.go

```diff
@@ -215,9 +215,11 @@ func makeMultiportDestinations(portsToIgnore []string) [][]string {
 }
 
 func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd, cmdOut io.Writer) error {
-	if strings.Contains(cmd.Path, "iptables") && firewallConfiguration.UseWaitFlag {
-		log.Info("'useWaitFlag' set: iptables will wait for xtables to become available")
-		cmd.Args = append(cmd.Args, "-w")
+	if firewallConfiguration.UseWaitFlag {
+		if strings.Contains(cmd.Path, "iptables") && !strings.Contains(cmd.Path, "save") {
```

Comment:
Testing the binary name for various substrings seems kinda brittle and confusing.  When we call into `executeCommand`, we know whether we're doing an `iptables` type command or an `iptables-save` type command.  (I think `iptables-save` is only used for doing the show all rule?).  Can we add the `-w` flag on line 90 instead, since at that point we know that we're calling an `iptables` command.  This way, we don't need to depend on how the binary is named.

Reply:
Great suggestion! Moved it up to line 90. Also, we'll log whether `useWaitFlag` is enabled earlier in the execution flow so we don't log it out every time we want to exec a command.